### ISSUE TITLE
Testimonials section - thoughts?

### ISSUE THREAD

**Issue by shymega** [chineham-repair-cafe/site-production#7]
The current template has, commented out, a 'testimonials section'.

I believe we've had, in the past, emails with positive comments, and even a letter to the local news publication.

If we could collate a list of testimonials somewhere, but with permission from the authors - and I think, for privacy, we should keep to first names or initials, this would be very useful for the website.

**Comment by sibo999** [comment]
Lets add a few in to staging. Any email we've got is fine.
One for Dom

**Comment by shymega** [comment]
Sure, assigned to me. I'll ask in the admin group if we have any email testimonials.

**Comment by jgclark** [comment]
Steve should have some written testimonials, as I added a section to the form used in Nov. Also Amy had a testimonial emailed in, from memory.

**Comment by shymega** [comment]
Yes, I'll ask Steve later on today for sure. He mentioned a Gazette letter in the WA as well - what are your thoughts on including that?
